source/code/source/playfab/PlayFabIOSHttpPlugin.mm

```diff
@@ -268,7 +268,14 @@ void Cancel()
                                                                        NSString* body = [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding];
 
                                                                        CallRequestContainer& requestContainer = request->RequestContainer();
-                                                                       requestContainer.responseString = [body UTF8String];
+                                                                       if(body != nil)
+                                                                       {
+                                                                        requestContainer.responseString = [body UTF8String];
+                                                                       }
+                                                                       else
+                                                                       {
+                                                                           requestContainer.responseString = "PlayFabIOSHttpPlugin Failed to parse body";
```

Comment:
I think something like "PlayFabIOSHttpPlugin Failed to parse body as it is nil" would be even more informative